src/js/control-bar/volume-panel.js

```diff
@@ -0,0 +1,89 @@
+/**
+ * @file volume-control.js
+ */
+import Component from '../component.js';
+import checkVolumeSupport from './volume-control/check-volume-support';
+import {isPlain} from '../utils/obj';
+
+// Required children
+import './volume-control/volume-control.js';
+import './mute-toggle.js';
+
+/**
+ * A Component to contain the MuteToggle and VolumeControl so that
+ * they can work together.
+ *
+ * @extends Component
+ */
+class VolumePanel extends Component {
+
+  /**
+   * Creates an instance of this class.
+   *
+   * @param {Player} player
+   *        The `Player` that this class should be attached to.
+   *
+   * @param {Object} [options={}]
+   *        The key/value store of player options.
+   */
+  constructor(player, options) {
+
+    // pass the inline option down to the VolumeControl if
+    // the VolumeControl is on.
+    if (options.inline === false && (!options.volumeControl || isPlain(options.volumeControl))) {
+      options.volumeControl = options.volumeControl || {};
+      options.volumeControl.inline = options.inline;
```

Comment:
maybe we should translate the `inline` option to vertical here instead of even at the volume control level?